sys/posix/pthread/pthread_attr.c

```diff
@@ -158,3 +158,21 @@ int pthread_attr_setstacksize(pthread_attr_t *attr, size_t stacksize)
     attr->ss_size = stacksize;
     return 0;
 }
+
+int pthread_attr_getstack(pthread_attr_t *attr, void **stackaddr, size_t *stacksize)
+{
+    *stackaddr = attr->ss_sp;
+    *stacksize = attr->ss_size;
+    if (*stacksize == 0) {
+        /* FIXME: the standard says that this function should return the default
+        * value if no explicit value was set. */
```

Comment:
So `THREAD_STACKSIZE_DEFAULT`?

Reply:
I am not sure about this comment. I copied it over from the implementation of `pthread_attr_getstacksize`.

This PR is really only a merge of these 2 functions for get and for set.

Reply:
The code line is 151 and was added in #1017